driver/src/device/configuration/device_configuration.cpp

```diff
@@ -64,6 +69,9 @@ nlohmann::ordered_map<std::string, std::variant<int>> GetAlphaEncodingConfigurat
 
   result["max_analog_value"] = vr::VRSettings()->GetInt32(k_alpha_encoding_settings_section, "max_analog_value");
 
+  if (std::get<int>(result["max_analog_value"]) == 0)
+      logger.Log(og::kLoggerLevel_Error, "max_analog_value is set to zero. This will cause errors, and your glove probably won't work.");
```

Comment:
didn't realise that this logger could be used in the driver as well! I think probably best to change this to DriverLog (as it's not part of the server)

```suggestion
      DriverLog("max_analog_value is set to zero. This will cause errors, and your glove probably won't work.");
```

Reply:
:0 lemme test it out